tools/gen_selectedops.py

```diff
@@ -74,7 +74,9 @@
     ],
     "OCOS_ENABLE_DLIB": [
         "Inverse",
-        "StftNorm"
+        "StftNorm",
+        "DecodeImage",
+        "EncodeImage"
```

Comment:
Both OCOS_ENABLE_VISION and OCOS_ENABLE_DLIB are required because libjpeg and libpng C source files are in _dep/dlib folder.

Reply:
BTW, enable_dlib doesn't introduce more footprint since we compile the required C++ files directly without any CMake dependencies. And if we could have native implementations for encode_image.cc, like decode_image did by @skyline75489, then libjpeg and etc can fully removed from iOS build.

Reply:
If we have enough test case covering the encoding scenarios, I think it should be possible to add native encoding capability on Windows & Apple platforms, just as their decoding counterpart in #808

Reply:
> If we have enough test case covering the encoding scenarios, I think it should be possible to add native encoding capability on Windows & Apple platforms, just as their decoding counterpart in #808

There are some Python test cases covered this operator, like this one: https://github.com/microsoft/onnxruntime-extensions/blob/5104bb989745dfb12b384add07d6886b504c25cd/test/test_tools_add_pre_post_processing_to_model.py#L248. We can add some native C++ test to replace this one: https://github.com/microsoft/onnxruntime-extensions/blob/main/test/shared_test/test_ortops_vision.cc too.